axiom/tests/test_log_events.c

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "nr_axiom.h"
+
+#include <stddef.h>
+
+#include "nr_log_events.h"
+#include "util_random.h"
+#include "util_strings.h"
+
+#include "tlib_main.h"
+
+#define LOG_LEVEL "INFO"
+#define LOG_MESSAGE "this is a test log error message"
```

Comment:
I'm curious what happens if there are things like formatting characters in a log message? Does it get sanitized out somewhere down the line? 
For example, if someone has a log message with a `'\n'` character, or `'\0'`, or `'%s'`. 
Maybe here would be a good place to test those kinds of messages?

Reply:
@bduranleau-nr It's a good idea. However I think a better place for this type of test would be either log_event `set_message` and `get_event_json` API, which log_events API rely on, or integration test which exercise all agent's interfaces that could have been impacted by special characters.